pacai/agents/ghost/directional.py

```diff
@@ -1,5 +1,6 @@
 from pacai.agents.ghost.base import GhostAgent
 from pacai.core.actions import Actions
+from pacai.util import probability
```

Comment:
Move this import down one to keep the imports sorted.

Reply:
Isn't it already sorted by decending length?

Reply:
Lexicographic order, not length.